src/Gateway.php

```diff
@@ -170,6 +166,13 @@ public function setUsername(string $username): self
         return $this;
     }
 
+    public function setTestMode($testMode): self
```

Comment:
This function is not covered, could you add a unit test for it? See `testInitialisationWillSetCorrectBaseUrl` as example